script/options.js

```diff
@@ -8,6 +8,7 @@ const environments = require('./constants/environments');
 const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'buildtype', type: String, defaultValue: environments.DEVELOPMENT },
   { name: 'brand-consolidation-enabled', type: Boolean, defaultValue: false },
+  { name: 'frontend-metrics-enabled', type: Boolean, defaultValue: false },
```

Comment:
Great question. I think for now limiting it to a passed argument is ok until we've done some more testing in a production-like environment.

Reply:
I don't think we want the separate command line option here. Brand consolidation does that because we need to build parallel environments. In this case, I think we just want to keep it turned off in production? Typically we check `__BUILDTYPE__` in code to do that kind of gating.

Reply:
Yeah agree with @jbalboni. This seems like a feature that should be toggled based on `__BUILDTYPE__`.